web/src/app/alerts/pages/AlertDetailPage.js

```diff
@@ -39,26 +39,23 @@ const query = gql`
   }
 `
 
-export default class AlertDetailPage extends Component {
-  render() {
-    return (
-      <Query
-        query={query}
-        variables={{ id: this.props.match.params.alertID }}
-        pollInterval={POLL_INTERVAL}
-      >
-        {({ loading, error, data, startPolling }) => {
-          if (!data && loading) return <Spinner />
-          if (error) {
-            startPolling(POLL_ERROR_INTERVAL)
-            return <GenericError error={error.message} />
-          }
+function AlertDetailPage(props) {
+  const { loading, error, data, startPolling } = useQuery(query, {
+    variables: { id: props.match.params.alertID },
+    pollInterval: POLL_INTERVAL,
```

Comment:
I think this and `startPolling` can be removed now that we're using `useQuery` it should be the default now